tests/9-soc/picosoc/CMakeLists.txt

```diff
@@ -7,28 +9,28 @@ add_file_target(FILE spimemio.v SCANNER_TYPE verilog)
 
 add_file_target(FILE picosoc_noflash.v SCANNER_TYPE verilog)
 
-add_file_target(FILE hx8kdemo.v SCANNER_TYPE verilog)
-add_file_target(FILE hx8kdemo_tb.v SCANNER_TYPE verilog)
-add_file_target(FILE hx8k-b-evn.pcf)
+get_target_property_required(PYTHON3 env PYTHON3)
+get_target_property(PYTHON3_TARGET env PYTHON3_TARGET)
 
-add_file_target(FILE basys3_demo.v SCANNER_TYPE verilog)
-add_file_target(FILE basys3_demo_no_roi.v SCANNER_TYPE verilog)
-add_file_target(FILE basys3_demo_tb.v SCANNER_TYPE verilog)
-add_file_target(FILE basys3.pcf)
-add_file_target(FILE basys3.sdc)
+file(GLOB HEX_FILES *.hex)
```

Comment:
Do not use GLOB

Reply:
What's wrong about using GLOB? I didn't wanted to hard code names of HEX files.

Reply:
We briefly discussed why GLOB should not be used.  Please remove it's usage.

Reply:
Ok, I've fixed that.